### PR TITLE
Remove short phrases from the game description

### PR DESCRIPTION
**Pull request by NeatNit**
Update the game description to no longer state that short phrases are possible guesses. Further down, explicitly declare that the secret word will not be a pair of words, implicitly this applies to guesses too.

That explicit info is further down because I think this minor nitpick of what counts as a word should not be front and center - it pretty much goes without saying, and only worth stating as a disclaimer.

Solves #38.

### CODE REVIEW

**Comment by NeatNit** [comment]
I made this change using GitHub's web UI. I've noticed that my quote marks (ASCII " Quotation Mark) are different from yours (Unicode U+05F4 ״ Hebrew Punctuation Gershayim). Gershayim should only be used in acronyms, for example צה״ל, not as quotation marks. Should I update that as well?

Also note that GitHub decided to add the missing newline at the end of the file. This should not cause any problems, just FYI :)

File: static/menu.html [hunk 1]
```diff
@@ -30,8 +30,8 @@
         <div class="dialog-content" id="rules-content">
             <h3 id="rules-heading">נחשו את המילה הסודית</h3>
             <p>
-            אפשר לנחש מילה או ביטוי קצר.
-            המילה הסודית יכולה להיות בכל אורך, אבל תהיה מילה בודדת (ולא ביטוי).
+            אפשר לנחש כל מילה בעברית.
```
Comment (ishefi) [inline static/menu.html:33]: Thanks!

How about adding an elaboration regarding what "בעברית" means? The secret word in Hebrew-Semantle #4 was "דנטלי" and many people didn't consider it as a Hebrew word (and there might be such words in the future)

Comment (NeatNit) [reply static/menu.html:33]: That's an interesting point... I'll try to mention it while staying short and to the point :)

Comment (NeatNit) [reply static/menu.html]: I brought this up in conversion. My dad suggested to change it to "any word in the Hebrew lexicon" (כל מילה בלקסיקון העברי or באוצר המילים בעברי). Either way, my parents didn't like my suggested phrasing (בעברית או לועזית), so perhaps you shouldn't merge yet!

**Pull request closed without merging**